specs/carves.table

```diff
@@ -0,0 +1,17 @@
+table_name("carves")
+description("Forensic Memory Carves")
+schema([
+    Column("time", BIGINT, "time at which the carve was kicked off"),
+    Column("sha256", TEXT, "A sha256 sum of the carved archive"),
+    Column("size", INTEGER, "Size of the carved archive"),
+    Column("path", TEXT, "The path of the requested carve"),
+    Column("status", TEXT, "Status of the carve, can be STARTING, PENDING, SUCCESS, or FAILED"),
```

Comment:
If you change to an enum for return codes, the verbiage here will need to be updated.